### ISSUE TITLE
Get ssid of connected WiFi

### ISSUE THREAD

**Issue by Martinn2** [ionic-team/capacitor-plugins#762]
### Plugin

Network

### Description

It would be great to get SSID of WiFi network device is connected to.

### Platform(s)

- Android
- iOS

**Comment by Darren777** [comment]
I strongly agree, this would be great. It is currently possible when using Cordova and the WiFiWizard2 plugin

**Comment by FranciskoNeves** [comment]
Adding this feature to network plugin would be great.

**Comment by eric-horodyski** [comment]
This thread has been inactive for some time, and we’re cleaning up older items to keep the backlog focused and manageable.

Details on the effort:
[https://ionic.io/blog/keeping-capacitors-backlog-healthy](https://ionic.io/blog/keeping-capacitors-backlog-healthy)

If this is still relevant today, feel free to open a new issue with current information and we’ll review it.
